src/main/java/ru/practicum/shareit/request/storage/RequestRepository.java

```diff
@@ -0,0 +1,14 @@
+package ru.practicum.shareit.request.storage;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import ru.practicum.shareit.request.model.Request;
+
+import java.util.List;
+
+public interface RequestRepository extends JpaRepository<Request, Integer> {
+    List<Request> findByRequestorIdOrderByCreatedDesc(int ownerId);
+
+    Page<Request> findByRequestorIdIsNot(int userId, Pageable pageable);
```

Comment:
Можно сразу ожидать список, а не объект `Page`, так как мы не используем его методы. Проще сразу будет работать со списком, а не с оберткой пагинации
```suggestion
    List<Request> findByRequestorIdIsNot(int userId, Pageable pageable);
```
Поправь, пожалуйста, для всех репозиториев

Reply:
Исправлено.